.github/codecov.yml

```diff
@@ -0,0 +1,18 @@
+coverage:
+  status:
+    project:
+      default:
+        # basic
+        target: auto
+        threshold: 0%
```

Comment:
0% ?

Reply:
Target is auto (don't drop below head branch aka master) 

threshold is % below the target that is allowed, so 0 meaning you can't lose coverage